react-frontend/src/pages/Home.js

```diff
@@ -38,8 +39,10 @@ class Home extends Component {
                       <span className="tank-title">Tank{tank.id}</span>
                     </div>
                   </center>
+                  {this.props.overflowAlarm(tank.value, tank.id)}
```

Comment:
Error in this line:
TypeError: this.props.overflowAlarm is not a function

Reply:
Still, this error is not fixed.

Reply:
@ArpitKotecha can I send you a DM on Slack?

Reply:
@IuliaElizaS I'm not on slack. @vinitshahdeo Please tag someone who can help.